README.md

```diff
@@ -39,7 +39,10 @@ of the configuration keys by scope:
 ### Global
 Contains configuration globally shared by all runners.
 
-* `labelSelector`: Label used to select services to mirror
+* `globalSvcLabelSelector`: Labels used to select global services
+* `globalSvcTopologyLabel`: Labels used to instruct controller to try utilising
```

Comment:
isn't this a selector too? (of services that want topology aware routing)

Reply:
pff difficult to define, as this works only on top of the above selector to signal to the operator that a service wants to try utilising topology aware hints. Also, it is renamed as `globalSvcRoutingStrategyLabel` in the latest commit